hist/histv7/test/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 # For the licensing terms see $ROOTSYS/LICENSE.
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
-if(MSVC AND MSVC_VERSION GREATER_EQUAL 1924)
+if(MSVC AND MSVC_VERSION GREATER_EQUAL 1923)
   # FIXME: using /O2 compiler flag prevent the followin error when building in debug mode:
   # axis.obj : fatal error LNK1179: invalid or corrupt file: duplicate COMDAT
   # '??$?8DU?$char_traits@D@std@@@__ROOT@experimental@std@@YA_NV?$basic_string_view@DU?$char_traits@D@std@@@012@0@Z'
```

Comment:
Is there a corresponding MSVC bug report?

Reply:
Nope, not sure if it's coming from MSVC, and didn't try to come with a simple reproducer

Reply:
fair enough ...